pkg/cache/cluster.go

```diff
@@ -223,9 +334,12 @@ func (c *clusterCache) GetAPIGroups() []metav1.APIGroup {
 	return c.apiGroups
 }
 
-func (c *clusterCache) replaceResourceCache(gk schema.GroupKind, resourceVersion string, objs []unstructured.Unstructured, ns string) {
-	info, ok := c.apisMeta[gk]
-	if ok {
+func (c *clusterCache) replaceResourceCache(gk schema.GroupKind, resourceVersion string, objs []unstructured.Unstructured, ns string) error {
+	info, err := c.LoadApisMeta(gk)
+	if err != nil {
+		return err
+	}
+	if info != nil {
```

Comment:
This `if` can be inverted to avoid so much indentation. https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow